src/SFA.DAS.AANHub.Data/Repositories/AttendancesReadRepository.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.EntityFrameworkCore;
+using SFA.DAS.AANHub.Data.Configuration;
+using SFA.DAS.AANHub.Domain.Entities;
+using SFA.DAS.AANHub.Domain.Interfaces.Repositories;
+
+namespace SFA.DAS.AANHub.Data.Repositories;
+
+[ExcludeFromCodeCoverage]
+public class AttendancesReadRepository : IAttendancesReadRepository
+{
+    private readonly AanDataContext _aanDataContext;
+
+    public AttendancesReadRepository(AanDataContext aanDataContext) => _aanDataContext = aanDataContext;
+
+    public async Task<Attendance?> GetAttendance(Guid calendarEventId, Guid memberId) =>
```

Comment:
Move this method to write repository as you are not using `AsNoTracking`

Reply:
Moved